services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -852,22 +854,47 @@ export class NoticeOfWorkApplication extends Component {
                 <LoadingWrapper condition={this.state.isTabLoaded}>
                   <div className={this.renderFixedHeaderClass()}>
                     <div className="inline-flex">
-                      <h2 className="padding-md">
-                        Referral
-                        <CoreTooltip
-                          title="This page allows you to identify and download the files that need to be included in the referral package.
+                      <h2 className="tab-title">
+                        <Popover
+                          placement="topLeft"
+                          content="This page allows you to identify and download the files that need to be included in the referral package.
             You may track progress on the E-Referrals website.
-            When responses are receives you can upload them by clicking on “Add Reviewer”
-            Finish this stage by clicking on “Complete Process” when all responses have been received.
-            If you need to make changes later, click “Resume Referral process”"
-                        />
+            When responses are receives you can upload them by clicking on “Add Referral”
```

Comment:
Can we make these quotes "normal" and just escape them instead? So `”` to `\"`?

Reply:
eagle eyes 🦅 
fixed